### PR TITLE
Add some missing flushes in configure.

### PR DESCRIPTION
**Pull request by maximedenes**
Some messages were sometimes not printed because of the missing flushes.
We use a generic combinator suggested by Emilio.

### CODE REVIEW

File: configure.ml [hunk 1]
```diff
@@ -22,8 +22,10 @@ let distributed_exec = ["coqtop";"coqc";"coqchk";"coqdoc";"coqworkmgr";
 let verbose = ref false (* for debugging this script *)
 
 (** * Utility functions *)
-
-let die msg = eprintf "%s\nConfiguration script failed!\n" msg; exit 1
+let cfprintf oc = kfprintf (fun oc -> fprintf oc "@\n%!") oc
```
Comment (ejgallego) [inline configure.ml]: "@\n" syntax is Format-only. Either use `Format` of switch to regular "\n"

Comment (maximedenes) [reply configure.ml]: Fixed, thanks!

**Comment by maximedenes** [comment]
@ejgallego does it now look good to you?

**Comment by ejgallego** [comment]
Yes it seems OK.

**Pull request merged**